src/app/invalid-session/page.tsx

```diff
@@ -0,0 +1,25 @@
+'use client';
+
+import content from '@content/content';
+import { BackLink } from 'nhsuk-react-components';
+
+const { invalidSessionPageContent } = content.pages;
+
+export default function InvalidSessionPage() {
+  return (
+    <div className='nhsuk-grid-row' data-testid='page-content-wrapper'>
+      <h1 className='nhsuk-heading-xl' data-testid='page-heading'>
+        {invalidSessionPageContent.pageHeading}
+      </h1>
+
+      <p>{invalidSessionPageContent.description}</p>
+
+      <BackLink
+        href={invalidSessionPageContent.backLink.path}
+        className='nhsuk-u-margin-bottom-7 nhsuk-u-margin-left-3'
+      >
+        {invalidSessionPageContent.backLink.text}
```

Comment:
back link at the bottom is not how we've set up our other pages - is this intentional?

Reply:
I'm awaiting feedback from Jake.  This is not actually a "back" action because your previous page was invalid, hence the error.  It's just a convenient styling.

Reply:
Updated following feedback.